SECP-service/SECP.yml

```diff
@@ -4,11 +4,9 @@ secretKey: o5KnhuFU8q8bo6qPxG2Z75q3yLlNiVCSOUd0vqbv6X8E8QLUSbP6Uxn7hn2678NZ3sQ1Z
 # Database settings.
 database:
   driverClass: com.mysql.jdbc.Driver
-  url: jdbc:mysql://localhost:3306/secp?createDatabaseIfNotExist=true
-  user: root
-  password:
-  minSize: 8
-  maxSize: 200
+  url: jdbc:mysql://aaxtpk0cgefevd.cpppkhaefajm.us-east-2.rds.amazonaws.com:3306/ebdb
+  user: secp
```

Comment:
Im assuming including our credentials here are mandatory right?

Reply:
To connect to the database, we need this. I know we should not be including this on the github page. I just don't want to be writing this everytime i want to deploy or work on the app.